### PR TITLE
Added login page

### LINKED ISSUE #12: Added login page
Added login page
created a footer
Login page can be accessed by clicking on login button

### PR DESCRIPTION
**Pull request by Raghav888**
Added login page
created a footer
Login page can be accessed by clicking on login button

### CODE REVIEW

**Review by rushikesh3299 — APPROVED** [review]
I liked that for footer & some other things you have used styling from old projects.
Also delete login.css if you don't need it.

File: src/components/loginform/Loginform.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+            <input
+              id="email"
+              class="text-box text-box-email mantra-textbox-classic mantra-highlight-box"
+              type="email"
+              placeholder=" xyz@email.com "
+              required
+            />
+          </div>
+          <div class="login-input">
+            <label for="password">Password: </label>
```
Comment (JantuDeb) [inline src/components/loginform/Loginform.js:23]: use `htmlFor` instead of `for`

File: src/components/loginform/Loginform.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+              id="password"
+              class="text-box mantra-textbox-classic mantra-highlight-box"
+              type="password"
+              placeholder="  ********** "
+              required
+            />
+          </div>
+          <div class="login-input-box">
+            <input type="checkbox" id="remember-me" />
+            <label for="remember-me"> Remember me</label>
```
Comment (JantuDeb) [inline src/components/loginform/Loginform.js:34]: same here

File: src/components/loginform/Loginform.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+            <div className="forgot-password"> Forgot password</div>
+          </div>
+          <button class="mantra-button mantra-primary-btn login-button">
```
Comment (JantuDeb) [inline src/components/loginform/Loginform.js:37]: `className`

File: src/components/loginform/Loginform.js [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+            <label for="password">Password: </label>
+            <input
+              id="password"
+              class="text-box mantra-textbox-classic mantra-highlight-box"
+              type="password"
+              placeholder="  ********** "
+              required
+            />
+          </div>
+          <div class="login-input-box">
```
Comment (JantuDeb) [inline src/components/loginform/Loginform.js:32]: className

File: src/components/loginform/Loginform.js [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+              type="email"
+              placeholder=" xyz@email.com "
+              required
+            />
+          </div>
+          <div class="login-input">
+            <label for="password">Password: </label>
+            <input
+              id="password"
+              class="text-box mantra-textbox-classic mantra-highlight-box"
```
Comment (JantuDeb) [inline src/components/loginform/Loginform.js:26]: className

File: src/components/loginform/Loginform.js [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+      <div class="login-box">
+        <div class="login-title">
+          <h3>Login</h3>
+        </div>
+        <form class="login-form" action="submit">
+          <div class="login-input">
+            <label for="email">Email Id: </label>
+            <input
+              id="email"
+              class="text-box text-box-email mantra-textbox-classic mantra-highlight-box"
```
Comment (JantuDeb) [inline src/components/loginform/Loginform.js:16]: className

**Review by JantuDeb — APPROVED** [review]
Added some suggestion

**Pull request merged**